### PR TITLE
User registrations confirmation Tests

### PR DESCRIPTION
**Pull request by kaybrian**
# Description
In this PR, the main aim was to confirm that a user has successfully registered after filling in the signup form

# Reason for the Test 
The main aim of doing this test is to fully confirm that a user has been registered in the database before starting to reference them on the project 

# Tests. 
all the tests in this PR has been  written in the tests.py file of the user app. the name for function of  the particular test is test_signup_form_sucess

### CODE REVIEW

File: contact_form_mail/settings.py [hunk 1]
```diff
@@ -73,18 +73,24 @@
...
-
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.environ.get('DB_NAME'),
-        'HOST': os.environ.get('DB_HOST', 'localhost'),
-        'USER': os.environ.get('DB_USER', 'postgres'),
-        'PASSWORD': os.environ.get('DB_PASSWORD', ''),
-        'PORT': os.environ.get('DB_PORT', '5432'),
+        'ENGINE': 'django.db.backends.sqlite3',
```
Comment (arthurarty) [inline contact_form_mail/settings.py]: revert this change. Also you can just create a postgres database.

Comment (kaybrian) [reply contact_form_mail/settings.py]: sorry i dont understand what you mean

File: user/tests.py [hunk 2]
```diff
@@ -28,3 +31,11 @@ def test_signup_view(self):
             view.func.__name__,
             SignupPageView.as_view().__name__
         )
+
+    def test_signup_form_sucess(self):
+        new_user = User.objects.create_user(self.username, self.email)
```
Comment (arthurarty) [inline user/tests.py:36]: This test is creating the user and then testing if the user it just created is in the database. 
But ideally it should be attempting to fill out the form and then check if by sending the form a user was created in the database.

**Review by arthurarty — CHANGES_REQUESTED** [review]
Need to make changes to the above.

File: contact_form_mail/settings.py (hunk 1 above)
Comment (arthurarty) [reply contact_form_mail/settings.py]: by revert I mean to undo the change. I see you are attempting to use an sql lite database. However, I want the app along with its functionality to be built and tested with Postgres. So uncomment the lines below starting with line 83.

File: user/tests.py (hunk 2 above)
Comment (kaybrian) [reply user/tests.py:36]: oooh thanks. i will look into it let change the tests
